### DISCUSSION TITLE
Mouse click and movement

### DISCUSSION THREAD

**Discussion by VivekaNinja** [KinesisCorporation/Adv360-Pro-ZMK#437] [Q&A]
Is there any chance to make mouse emulation of clicking and moving possible on my Adv360?
There even seems to be scrolling possible with zmk pull request 2027.
https://github.com/zmkfirmware/zmk/pull/2027
I am pretty unexperienced on zmk and am greatful for any hints.

**Comment by ghost** [comment]
+1
I have built my own branch with the functionality merged in but would be nice to have it in the main upstream and official.  Been waiting since my 360 arrived in 9/202x whenever it was released.

**Reply by VivekaNinja** [reply]
Cool. Would you share that branch please?

**Reply by ReFil** [reply]
That pull request is still getting frequently rebased and changed. Once it's merged in to upstream it will be implemented in the official 360 Pro repo

**Reply by VivekaNinja** [reply]
It seems to be merged now. Are there plans already to implement it?
